Overview.src.html

```diff
@@ -662,6 +662,15 @@ <h4 id=terminology-headers>Headers</h4>
 <span data-anolis-spec=html>environment settings object</span>).
 
 <p>A <span title=concept-request>request</span> has an associated
+<dfn title=concept-request-reserved-client data-export data-dfn-for=request>reserved client</dfn>
+(null or an <span data-anolis-spec=html>environment</span> or an
+<span data-anolis-spec=html>environment settings object</span>). Unless stated otherwise it is null.
+
+<p>A <span title=concept-request>request</span> has an associated
+<dfn title=concept-request-target-client-id data-export data-dfn-for=request>target client id</dfn>
+(null or a string). Unless stated otherwise it is null.
```

Comment:
Can't we just use the empty string here? It's usually better to not change data types if not necessary.